base/gcutils.jl

```diff
@@ -119,9 +119,9 @@ const GC_INCREMENTAL = 2
     GC.gc([full=true])
 
 Perform garbage collection. The argument `full` determines the kind of
-collection: A full collection (default) sweeps all objects, which makes the
-next GC scan much slower, while an incremental collection may only sweep
-so-called young objects.
+collection: a full collection (default) traverses all live objects (e.g. full mark)
```

Comment:
```suggestion
collection: a full collection (default) traverses all live objects (i.e. full mark)
```

or? I don't really understand the "e.g." usage here.